gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/catalog/GfxdSystemProcedures.java

```diff
@@ -2568,8 +2563,8 @@ public static String GET_SNAPSHOT_TXID() throws SQLException, StandardException
     }
     // tc commit will clear all the artifacts but will not commit actual txState
     // that should be committed in COMMIT procedure
-    tc.resetActiveTXState(true);
-    TXManagerImpl.getOrCreateTXContext().clearTXState();
+    tc.clearActiveTXState(true, true);
+    //tc.resetActiveTXState(true);
```

Comment:
@suranjan I thought that row buffer operations and column store ops used the same connection. Weren't the changes that added optional implicit connection argument to storeColumnBatch etc for that? Any specific reason to use different connections for row buffer inserts and column table inserts, and won't using the same for both make code cleaner and simpler?